yew-functional/src/lib.rs

```diff
@@ -1,12 +1,11 @@
-use std::borrow::Borrow;
 use std::cell::RefCell;
-use std::ops::DerefMut;
+
 use std::rc::Rc;
 use yew::html::AnyScope;
 use yew::{Component, ComponentLink, Html, Properties};
 
-mod use_context_hook;
-pub use use_context_hook::*;
+pub mod hooks;
+pub use hooks::*;
```

Comment:
Apart from the prelude, there should only be one public path to a given member, having both `yew-functional::hooks::use_state` and `yew-functional::use_state` is confusing.

I'm fine with either a public submodule or re-exporting all members though.

Reply:
Hooks is no longer public, and all hooks are re-exported.